river/expert/bandit.py

```diff
@@ -193,7 +198,8 @@ def _default_params(cls):
                 linear_model.LinearRegression(intercept_lr=.01)
             ],
             'metric': metrics.MSE(),
-            'reward_scaler': preprocessing.StandardScaler()
+            'reward_scaler': preprocessing.StandardScaler(),
+            'seed': 1
```

Comment:
No need to set the seed in the `default_params`. At least that's the current convention.

Reply:
Ok ! I thought it was necessary for the tests